src/main.js

```diff
@@ -2,24 +2,37 @@ import $ from 'jquery';
 import 'bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './css/styles.css';
-import { Catalog } from './js/course-catalog.js';
+// import { Catalog } from './course-catalog.js';
 
```

Comment:
Do y'all need the commented out code? If not, can we delete it from the repo file so that we minimize any potential merge conflicts?

Reply:
also the console.logs unless you still need them!!

Reply:
also the console.logs unless you still need them!!